cpp/src/groupby/common/aggregation_requests.cpp

```diff
@@ -105,7 +105,7 @@ gdf_column* compute_average(gdf_column sum, gdf_column count, cudaStream_t strea
   return avg;
 }
 
-table compute_original_requests(
+table compute_original_aggregations(
```

Comment:
I thought @jrhemstad argued pretty convincingly to stay with "requests".

Reply:
Original request: the original set of potentially compound aggregations which are used to compute the compound aggregations using the result of simple_aggregation_request.
 
The remaining are just `ordered_aggregations`. So I renamed the function to just `compute_ordered_aggregations`. To refer that if there are "ordered" aggregation requests like MEDIAN, QUANTILE, compute them.